libfuse/mount_test.go

```diff
@@ -1101,6 +1101,40 @@ func TestRemoveDirNotEmpty(t *testing.T) {
 	}
 }
 
+func TestRemoveFileWhileOpenSetEx(t *testing.T) {
```

Comment:
Because the DSL tests don't have support for holding {{Node}}s (or file handles) between operations -- each operation results in a new lookup.